mallard/url.py

```diff
@@ -0,0 +1,43 @@
+#
+# url.py
+#
+# mallard - DDG Instant Answers bot for the Programming Server
+# Copyright (c) 2017 Ammon Smith et al.
+#
+# mallard is available free of charge under the terms of the MIT
+# License. You are free to redistribute and/or modify it under those
+# terms. It is distributed in the hopes that it will be useful, but
+# WITHOUT ANY WARRANTY. See the LICENSE file for more details.
+#
+
+from typing import Optional
+
+import aiohttp
+
+async def try_follow_redirect(url: str, default: str = None) -> Optional[str]:
```

Comment:
why didn't pylint yell about this? IIRC it's 2 newlines before a function definition

Reply:
:/ I've never been a fan of the two lines thing. I don't think it looks super great.